packages/xchain-ethereum/src/client.ts

```diff
@@ -187,11 +185,11 @@ export default class Client implements XChainClient, EthereumClient {
    * @throws {"Phrase must be provided"}
    * Thrown if phrase has not been set before. A phrase is needed to create a wallet and to derive an address from it.
    */
-  getAddress = (): Address => {
-    if (!this.address) {
-      throw new Error('Phrase must be provided')
+  getAddress = (index = 0): Address => {
+    if (index < 0) {
+      throw new Error('index must be greater than zero')
     }
-    return this.address
+    return this.hdNode.derivePath(this.getFullDerivationPath(index)).address.toLowerCase()
```

Comment:
Why `toLowerCase`?

Reply:
i removed the old changeWallet() when i removed  the wallet as a an instance variable 
 that old call was using toLower() so i kept it for compatibility
```
 /**
   * Changes the wallet eg. when using connect() after init().
   *
   * @param {Wallet} wallet a new wallet
   * @returns {void}
   */
  private changeWallet = (wallet: ethers.Wallet): void => {
    this.wallet = wallet
    this.address = wallet.address.toLowerCase()
  }
```